spec/models/quiz_spec.rb

```diff
@@ -5,4 +5,20 @@
 
   it { should validate_presence_of(:name) }
   it { should validate_uniqueness_of(:name) }
+
+  it "answers if it has been completed by a student" do
+    user = FactoryGirl.create(:user_with_answered_quiz)
+    bad_user = FactoryGirl.create(:user)
+    quiz = Quiz.first
+
+    expect(quiz.completed_by_student?(user)).to eq(true)
+    expect(quiz.completed_by_student?(bad_user)).to eq(false)
+  end
+
+  it "returns a student's score" do
+    user = FactoryGirl.create(:user_with_answered_quiz)
+    quiz = Quiz.first
```

Comment:
For this test, I think it would be better if you create a quiz with questions, and then create answer submissions for the user.